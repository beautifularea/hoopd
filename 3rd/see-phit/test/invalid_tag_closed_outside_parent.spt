R"*(
<HTML>
  <DIV> 
    Tag closed outside parent
    <STRONG>
  </DIV>
  </STRONG>
</HTML>
)*"_html; 