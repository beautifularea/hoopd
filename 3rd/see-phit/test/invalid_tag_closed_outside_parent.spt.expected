Error() [with int ROW = 6; int COL = 5; WHAT = spt::Mismatched_Close_Tag]':
